src/prefect/engine.py

```diff
@@ -1642,7 +1643,7 @@ async def orchestrate_task_run(
         result_factory=result_factory,
         log_prints=log_prints,
     )
-
+    start_time = time.time()
```

Comment:
Was there any particular reason for using `time.time()` over `time.perf_counter()` here?

Reply:
not really, updated to use `perf_counter`